src/models/stores/exemplar-controller.ts

```diff
@@ -48,6 +51,12 @@ export const BaseExemplarControllerModel = types
     setExemplarVisibility(key: string, isVisible: boolean) {
       if (self.db) {
         self.db.firebase.ref(self.firebasePath).child(`${key}/visible`).set(isVisible);
+        if (isVisible) {
+          // Notify user with a sticky note
+          const audience = AudienceModel.create({type: AudienceEnum.user, identifier: self.db.stores.user.id});
+          const message = "Nice work, you can now see a new example for this lesson:";
+          self.db.createSupport(createStickyNote(message, key), "", audience);
+        }
```

Comment:
Since in the next story the visibility of an exemplar might be triggered remotely by a teacher, you could change this so it is watching database for the change instead being triggered directly by `setExemplarVisibility`.  Perhaps it is best to wait for that story and then you can remove this code and have both paths go through same DB listener.